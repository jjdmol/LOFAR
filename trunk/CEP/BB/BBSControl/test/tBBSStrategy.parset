DataSet                  = "test.ms"        # name of Measurement Set

Strategy.Steps           = ["xyz1", "xyz2"] # steps
Strategy.Stations        = [ 0, 1, 2, 3 ]   # ID's of stations to use
Strategy.InputData       = "INDATA"         # MS input data column 
Strategy.Correlation.Selection = ALL        # one of AUTO, CROSS, ALL
Strategy.Correlation.Type = ["XX", "YY"]    # which (cross)correlations to use
Strategy.WorkDomainSize.Freq = 1e+6         # work domain size: f(Hz) 
Strategy.WorkDomainSize.Time = 10           # work domain size: t(s)
Strategy.Integration.Freq = 1;              # integration interval: f(Hz)
Strategy.Integration.Time = 0.1;            # integration interval: t(s)

BBDB.Host                = "127.0.0.1"      # hostname/ipaddr of BB DBMS
BBDB.Port                = 12345            # port used by BB DBMS
BBDB.DBName              = "blackboard"     # name of the BB database
BBDB.UserName            = "postgres"       # username for accessing the DBMS
BBDB.PassWord            = ""               # password for accessing the DBMS
BBDB.ParmDB.Name         = "parmdb"         # table containing (solved) params.
BBDB.ParmDB.Type         = "aips"           # "database" used to store params.
BBDB.ParmDB.Instrument   = "test.mep"       # instrument parameters (MS table)
BBDB.ParmDB.LocalSky     = "test.gsm"       # local sky parameters (MS table)

Step.xyz1.Steps          = ["sl1", "sl2"]
Step.xyz1.Selection.Station1 = [0, 0, 0, 1, 1, 2] # baselines to use 
Step.xyz1.Selection.Station2 = [0, 1, 2, 1, 2, 2] # (all if empty)
Step.xyz1.Sources        = ["3C343"]        # list of sources
Step.xyz1.InstrumentModel = ["Bandpass",  \ # Instrument model
                           "DirGain", "Phase"]

Step.xyz2.Steps          = ["sl1", "sl3"]
Step.xyz2.Selection.Station1 = [0, 0, 1]    # baselines to use
Step.xyz2.Selection.Station2 = [0, 1, 1]    # (all if empty)
Step.xyz2.Sources        = ["3C343"]        # sources in the source model
Step.xyz2.InstrumentModel = ["Bandpass"]    # Instrument model

Step.sl1.Selection.Station1 = [0, 1]        # baselines to use
Step.sl1.Selection.Station2 = [1, 2]        # (all if empty)
Step.sl1.Sources         = []               # list of sources
Step.sl1.Operation       = SOLVE          \ # one of SOLVE, SUBTRACT, CORRECT,
                                            # PREDICT, SHIFT, REFIT
Step.sl1.OutData         = "OUTDATA1"       # MS output data column
Step.sl1.Solve.MaxIter   = 10               # maximum number of iterations
Step.sl1.Solve.Epsilon   = 1e-7             # convergence threshold
Step.sl1.Solve.MinConverged = 0.95          # fraction that must have converged
Step.sl1.Solve.Parms     = ["PHASE:*"]      # names of solvable parameters
Step.sl1.Solve.ExclParms = [""]             # parameters excluded from solve
Step.sl1.Solve.DomainSize.Freq = 1000       # f(Hz)
Step.sl1.Solve.DomainSize.Time = 1          # t(s)
Step.sl1.InstrumentModel = ["Bandpass", "Phase"]

Step.sl2.Selection.Station1 = []            # baselines to use
Step.sl2.Selection.Station2 = []            # (all if empty)
Step.sl2.Sources         = []               # list of sources
Step.sl2.Operation       = CORRECT        \ # one of SOLVE, SUBTRACT, CORRECT,
                                            # PREDICT, SHIFT, REFIT
Step.sl2.OutData         = "OUTDATA2"       # MS output data column
Step.sl2.Solve.MaxIter   = 10               # maximum number of iterations
Step.sl2.Solve.Epsilon   = 1e-6             # convergence threshold
Step.sl2.Solve.MinConverged = 0.85          # fraction that must have converged
Step.sl2.Solve.Parms     = ["GAIN:*"]       # names of solvable parameters
Step.sl2.Solve.ExclParms = ["PHASE:*"]      # parameters excluded from solve
Step.sl2.Solve.DomainSize.Freq = 500        # f(Hz)
Step.sl2.Solve.DomainSize.Time = 2          # t(s)
Step.sl2.InstrumentModel = ["DirGain", "Phase"]

Step.sl3.Selection.Station1 = []            # baselines to use
Step.sl3.Selection.Station2 = []            # (all if empty)
Step.sl3.Sources         = []               # list of sources
Step.sl3.Operation       = PREDICT        \ # one of SOLVE, SUBTRACT, CORRECT,
                                            # PREDICT, SHIFT, REFIT
Step.sl3.OutData         = "OUTDATA3"       # MS output data column
Step.sl3.Solve.MaxIter   = 10               # maximum number of iterations
Step.sl3.Solve.Epsilon   = 1e-5             # convergence threshold
Step.sl3.Solve.MinConverged = 0.98          # fraction that must have converged
Step.sl3.Solve.Parms     = ["GAIN:*", "PHASE:*"] # names of solvable parameters
Step.sl3.Solve.ExclParms = [""]             # parameters excluded from solve
Step.sl3.Solve.DomainSize.Freq = 250        # f(Hz)
Step.sl3.Solve.DomainSize.Time = 4          # t(s)
Step.sl3.InstrumentModel = ["Phase"]
