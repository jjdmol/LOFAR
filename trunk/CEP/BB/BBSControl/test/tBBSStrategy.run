#!/bin/sh
\cp $srcdir/tBBSControl.parset .
./tBBSStrategy
STATUS=$?
\rm -f tBBSControl.parset
exit $STATUS
