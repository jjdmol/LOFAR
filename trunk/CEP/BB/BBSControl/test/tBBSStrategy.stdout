
Measurement Set: test.ms
Blackboard database:
. Host: dop50.nfra.nl
. Port: 5432
. DBName: bbs
. Username: postgres
. Password: 
Parameter database:
. Instrument table: test.mep
. Local sky table: test.gsm
. History table: test.his
Strategy:
. Input data: INDATA
. Region of interest: [1.175e+09,1.176e+09,1000,3000]
. Domain size:
. . Bandwidth: 1e+06 (Hz)
. . Time interval: 10 (s)
. Correlation:
. . Selection: ALL
. . Type: [XX,YY]
. Integration:
. . Delta frequency: 1 (Hz)
. . Delta time: 0.1 (s)
. Stations: [CS*,RS1*,RS2*,DE*]
. Step: xyz1
. . Full name: xyz1
. . Baselines:
. . . Station1: [CS*,CS*,CS*,RS1*,RS1*,RS2*]
. . . Station2: [CS*,RS1*,RS2*,RS1*,RS2*,RS2*]
. . Correlation:
. . . Selection: CROSS
. . . Type: [XX,XY,YX,YY]
. . Integration:
. . . Delta frequency: 2 (Hz)
. . . Delta time: 0.5 (s)
. . Sources: [3C343]
. . Extra sources: [M81]
. . Instrument models: [Bandpass,DirGain,Phase]
. . Step: sl1
. . . Full name: xyz1.sl1
. . . Baselines:
. . . . Station1: [CS*,RS1*]
. . . . Station2: [RS1*,RS2*]
. . . Correlation:
. . . . Selection: CROSS
. . . . Type: [XX,XY,YX,YY]
. . . Integration:
. . . . Delta frequency: 2 (Hz)
. . . . Delta time: 0.5 (s)
. . . Sources: [3C343]
. . . Extra sources: [M81]
. . . Instrument models: [Bandpass,Phase]
. . . Output data: OUTDATA1
. . . Solve: 
. . . . Max nr. of iterations: 10
. . . . Convergence threshold: 1e-07
. . . . Min fraction converged: 0.95
. . . . Solvable parameters: [PHASE:*]
. . . . Excluded parameters: []
. . . . Domain size:
. . . . . Bandwidth: 1000 (Hz)
. . . . . Time interval: 1 (s)
. . Step: sl2
. . . Full name: xyz1.sl2
. . . Baselines:
. . . . Station1: [CS*,CS*,CS*,RS1*,RS1*,RS2*]
. . . . Station2: [CS*,RS1*,RS2*,RS1*,RS2*,RS2*]
. . . Correlation:
. . . . Selection: CROSS
. . . . Type: [XX,XY,YX,YY]
. . . Integration:
. . . . Delta frequency: 2 (Hz)
. . . . Delta time: 0.5 (s)
. . . Sources: [3C343]
. . . Extra sources: [M81]
. . . Instrument models: [DirGain,Phase]
. . . Output data: OUTDATA2
. Step: xyz2
. . Full name: xyz2
. . Baselines:
. . . Station1: [CS*,CS*,RS1*]
. . . Station2: [CS*,RS1*,RS1*]
. . Correlation:
. . . Selection: NONE
. . . Type: []
. . Integration:
. . . Delta frequency: 0 (Hz)
. . . Delta time: 0 (s)
. . Sources: [3C343]
. . Extra sources: []
. . Instrument models: [Bandpass]
. . Step: sl1
. . . Full name: xyz2.sl1
. . . Baselines:
. . . . Station1: [CS*,RS1*]
. . . . Station2: [RS1*,RS2*]
. . . Correlation:
. . . . Selection: NONE
. . . . Type: []
. . . Integration:
. . . . Delta frequency: 0 (Hz)
. . . . Delta time: 0 (s)
. . . Sources: [3C343]
. . . Extra sources: []
. . . Instrument models: [Bandpass,Phase]
. . . Output data: OUTDATA1
. . . Solve: 
. . . . Max nr. of iterations: 10
. . . . Convergence threshold: 1e-07
. . . . Min fraction converged: 0.95
. . . . Solvable parameters: [PHASE:*]
. . . . Excluded parameters: []
. . . . Domain size:
. . . . . Bandwidth: 1000 (Hz)
. . . . . Time interval: 1 (s)
. . Step: sl3
. . . Full name: xyz2.sl3
. . . Baselines:
. . . . Station1: [CS*,CS*,RS1*]
. . . . Station2: [CS*,RS1*,RS1*]
. . . Correlation:
. . . . Selection: NONE
. . . . Type: []
. . . Integration:
. . . . Delta frequency: 0 (Hz)
. . . . Delta time: 0 (s)
. . . Sources: [3C343]
. . . Extra sources: []
. . . Instrument models: [Phase]
. . . Output data: OUTDATA3
